templates/hexagonal-spring-boot/domain-api/src/main/java/packageName/domain/model/Example.java

```diff
@@ -9,7 +9,8 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-public class ExamplePoemInfo {
+public class Example {
 
-  private String poem;
+  private Long id;
```

Comment:
Should this class be part of rest-adapter? Later on while adding swagger documentation you have to again move it to rest-adapter considering domain purity.

Reply:
It is in the right place it is a pojo and unique identifier of the domain and should be part of the domain

Reply:
I did not understand. Domain should be pure and it should be pure java without any dependency. If I want to configure this pojo with swagger documentation our domain will ultimately have swagger dependency. Am I missing something?

Reply:
I dont understand where is the swagger dependency ??

Reply:
As per our discussion  as you mentioned we are using spring api doc and not swagger and you do not want to configure pojos to that detailed level for documentation. It all make sense. 
I approve it.